lib/shared/messages.js

```diff
@@ -32,7 +32,10 @@ module.exports = {
    */
   info: {
 
-    flashComplete: _.template('Your flash is complete!')
+    flashComplete: _.template([
+      '<%= imageBasename %> was written to',
```

Comment:
It's very context-sensitive, but here's GNOME 3. As you can see it gets truncated, but you can expand it by hovering over it.
![image](https://cloud.githubusercontent.com/assets/819413/26310957/8c86d14e-3efa-11e7-9e01-fecc49fc8630.png)

Maybe we can utilize the title space?